src/core/setState.js

```diff
@@ -1,5 +1,4 @@
-export function setState(store, newState, afterUpdateCallback) {
+export function setState(store, newState) {
```

Comment:
Disagree.
It will be a breaking change if we remove. and I don't see a benefit in removing in it.
If you don't mind reverting this then PR is good to go

Reply:
Ok, done